addons/l10n_jo_edi/__manifest__.py

```diff
@@ -10,7 +10,6 @@
     """,
     'depends': ['account_edi_ubl_cii', 'l10n_jo'],
     'data': [
-        'data/ubl_jo_templates.xml',
```

Comment:
Note that the file is still in the folder `data`. Do we really want to keep it?

Reply:
ooooh. I don't want to - I probably made a mistake when rebasing. Thanks so much for flagging it!!!